arbeitszeit_flask/dependency_injection/__init__.py

```diff
@@ -282,6 +263,14 @@ def provide_prefilled_draft_data_controller(
 
 
 class FlaskModule(PresenterModule):
+    @provider
+    def provide_request(self, request: FlaskRequest) -> Request:
+        return request
+
+    @provider
+    def provide_plan_summary_url_index(self, index: GeneralUrlIndex) -> UrlIndex:
```

Comment:
why is this called `provide_plan_summary_url_index`? isn't its purpose more general?